### ISSUE TITLE
Remove distinction between HTML5 and HTML4

### ISSUE THREAD

**Issue by r12a** [w3c/i18n-checker#10]
It doesn't make sense to report things like dir only accepts ltr and rtl for HTML4, but accepts ltr, rtl and auto for HTML5.

We should just check for HTML5 conformance.

Practically-speaking, an easy way to alter the code may be to just change $this->doc->html4 to $this->doc->html5.

I think we should probably also do the same for XHTML 1.0 and XHTML 1.1.

**Comment by r12a** [comment]
> I think we should probably also do the same for XHTML 1.0 and XHTML 1.1.

To be clearer, i'm proposing that we tailor the reports to just the distinction between pages served as HTML (html4, html5, xhtml1.0, polyglot) and pages served as XML (xhtml1.0, xhtml1.1, polyglot), and clarify in the description that we are more concerned with recommending best practices for creating web pages than evaluating conformance to a specific doctype.

We can still show the doc type of the page at the top right of the table, just not make such fine distinctions between the wording of errors, warnings and info items.

**Comment by r12a** [comment]
See the current tailorings at 
https://www.w3.org/International/quicktips/checker

**Comment by sideshowbarker** [comment]
> To be clearer, i'm proposing that we tailor the reports to just the distinction between pages served as HTML (html4, html5, xhtml1.0, polyglot) and pages served as XML (xhtml1.0, xhtml1.1, polyglot),

HTML5 can be served as XML, and in that case it’s not necessarily “polyglot”. If served as XML, an HTML5 document can, for example, use self-closing tag syntax—in particular for cases, like `<script src="foo"/>` where it makes a big difference. “polyglot” does not allow that, but HTML5-served-as-XML does.

Also worth noting is that https://www.w3.org/TR/html-polyglot/ never reached Rec and never will. It was retired as a Note, with a giant warning saying:

> This specification is no longer in active maintenance and the HTML Working Group does not intend to maintain it further.

Also HTML5 supersedes and obsoletes XHTML1.0 and XHTML1.1 in exacty the same way that HTML5 supersedes and obsoletes HTML4 (XHTML1.0 is simply HTML4 in XML syntax).

So in short the only current W3C specification for HTML is HTML5.

Given that I think that in 2016 it makes no more sense to give special handling to docs with XHTML1.0/XHTML1.0 doctypes or HTML4 doctypes than it does to give special handling to docs with HTML 3.2 doctypes.

Everything should instead just be handles as _HTML_—versionless—with the handling conforming to whatever the current HTML spec.

**Comment by r12a** [comment]
This is currently in progress. See the most recent commits as of the date of this comment.  There are some more commits to come shortly.

**Comment by r12a** [comment]
Done (across many commits).
